AmethystAPI/src/amethyst/Memory.h

```diff
@@ -8,6 +8,31 @@
 #include <vector>
 #include <windows.h>
 
+template <typename Ret, typename Type>
+typename std::conditional<std::is_const<Type>::value, std::add_const<Ret>, std::remove_const<Ret>>::type&
+direct_access(Type* type, size_t offset) {
```

Comment:
For naming consistency in the project, we are following PascalCase, `direct_access` would be better as `DirectAccess`